### ISSUE TITLE
Custom tiles servers

### ISSUE THREAD

**Issue by damianofalcioni** [peterLaurence/TrekMe#84]
Hi Peter,
would be great to have the possibility to customize the list of tile servers.
A simple syntax for the URLs could be "https://tile-a.openstreetmap.fr/hot/%z/%x/%y.png" and the information needed will only be the min and max allowed zoom level, a name for the map service and the URL (or better a list of URLs as many map services provide multiple DNS names like tile-a, tile-b,... ).

**Comment by 29jm** [comment]
I agree that it would great to have, as the app could work with many more country-specific maps. It's usually missing from mapping apps, or in some cases like CalTopo, it's possible to add custom layers but you can't download them for offline use.

I wanted to mention the WMTS (or WMS) standard for this. I know that for instance, official Japanese and Czech maps (equivalents of French IGN maps for these countries) are available through these protocols.
